packages/@o3r/rules-engine/src/services/rules-engine.service.ts

```diff
@@ -196,6 +200,10 @@ export class RulesEngineService implements OnDestroy {
     }));
   }
 
+  private registerOperatorFacts() {
+    this.upsertFacts([{ id: 'o3rCurrentTime', value$: currentTimeSubject$.asObservable() }]);
```

Comment:
The documentation `docs/rules-engine/facts.md` should contain the list of facts available per default